AGENTLESS.md

```diff
@@ -0,0 +1,64 @@
+# PingFederate Authentication API Agentless Integration
```

Comment:
would you please move this to "docs" folder (create it in your branch if it doesn't exist) and please rename it to "agentless.md". 

There's an example in master.